cmd/gen/dependabot/command.go

```diff
@@ -11,14 +11,7 @@ import (
 
 const (
 	name        = "dependabot"
-	description = `Generates GitHub Dependabot config (.github/dependabot.yml).
-
```

Comment:
I realised this breaks the formatting when using devtctl gen -- help 🤦. I changed it a bit, I hope it's still clear what it does

Reply:
You can add `Examples` to the `cobra.Command`.

Reply:
@kopiczko thanks that's what I'm looking for 👊